pkg/sloop/webserver/webfiles/sloop_ui.js

```diff
@@ -536,4 +536,13 @@ $(document).ready(function() {
     userDate.setMilliseconds(null);
     document.getElementById('selectedEndTime').value = userDate.toISOString().slice(0, -1);
 
+    $('#now').click(function(){
+        const resetNow = new Date();
+        resetNow.setMinutes(resetNow.getMinutes() - resetNow.getTimezoneOffset());
+        resetNow.setMilliseconds(null);
+        document.getElementById('selectedEndTime').value = resetNow.toISOString().slice(0, -1);
+        sessionStorage.clear();
```

Comment:
does `sessionStorage` contain any values other then `selectedEndTime`? If so, `clear()` may be destructive

Reply:
Yeah, We have another val called `setSelectedEndTime`, which is used to keep track when user select a time on date time picker. setSelectedEnd Time is set [here](setSelectedEndTime) and used [here](setSelectedEndTime). If clear() is a concern, it can be fixed by using remove() instead

Reply:
updated by using `removeItem()` other than `clear()`